### PR TITLE
Add telemetry BOND file for partner team.

### PR DESCRIPTION
**Pull request by msolivialuo**

### CODE REVIEW

File: ADAL/src/telemetry/ADTelemetryHttpEvent.m [hunk 1]
```diff
@@ -78,7 +79,23 @@ - (void)setHttpResponseMethod:(NSString*)method
 
 - (void)setHttpRequestQueryParams:(NSString*)params
 {
-    [self setProperty:AD_TELEMETRY_KEY_REQUEST_QUERY_PARAMS value:params];
+    if ([NSString adIsStringNilOrBlank:params])
+    {
+        return;
+    }
+    
+    NSArray *queryParams = [params componentsSeparatedByString:@"&"];
```
Comment (RPangrle) [inline ADAL/src/telemetry/ADTelemetryHttpEvent.m]: Please use +[NSDictionary adURLFormDecode:]

File: ADAL/src/utils/ADIpAddressHelper.m [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+// THE SOFTWARE.
+
+#import "ADIpAddressHelper.h"
+
+#import <ifaddrs.h>
+#import <arpa/inet.h>
+
+@implementation ADIpAddressHelper
+
++ (NSString *)adDeviceIpAddress
```
Comment (addev-ashish) [inline ADAL/src/utils/ADIpAddressHelper.m]: this should be in defaultEvent, why is this a separate class?

Comment (msolivialuo) [inline ADAL/src/utils/ADIpAddressHelper.m]: Discussed this with Ashish offline. The code to get IP address doesn't belong to telemetry event class, which should contain collecting telemetry data.

File: documentation/telemetry/ADAL.BOND [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+   60: string Microsoft.ADAL.application_version;
+   
+   [Name("TokenRequestId")]
+   70: optional string Microsoft.ADAL.request_id;
+   
+   [Name("RequestCorrelationId")]
+   80: string Microsoft.ADAL.correlation_id;
+   
+   [Name("ClientDeviceSKU")]
+   90: string Microsoft.ADAL.x_client_SKU;
```
Comment (msolivialuo) [inline documentation/telemetry/ADAL.BOND]: Updated.

File: documentation/telemetry/ADAL.BOND [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+   
+   [Name("ClientADALVersion")]
+   100: string Microsoft.ADAL.x_client_Ver;
```
Comment (msolivialuo) [inline documentation/telemetry/ADAL.BOND]: Updated

**Pull request merged**